### PR TITLE
change UTC to Local

### PR DESCRIPTION
**Pull request by wkshare**
clean *.ts utc
clean *.go utc
add timeZone to dev.conf

### CODE REVIEW

File: cmd/bosun/conf/conf.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ import (
 type Conf struct {
 	Vars
 	Name            string        // Config file name
+        TimeZone        string        // Config timezone, default UTC
```
Comment (mjibson) [inline cmd/bosun/conf/conf.go]: Need to gofmt this file.

File: cmd/bosun/conf/conf.go [hunk 2]
```diff
@@ -363,6 +365,14 @@ func New(name, text string) (c *Conf, err error) {
 func (c *Conf) loadGlobal(p *parse.PairNode) {
 	v := c.Expand(p.Val.Text, nil, false)
 	switch k := p.Key.Text; k {
+        case "timeZone":
+                _, err := time.LoadLocation(v)
+                if err != nil {
+                        c.error(err)
+                        c.TimeZone = "UTC"
+                } else {
+
```
Comment (mjibson) [inline cmd/bosun/conf/conf.go]: Missing an action here. If UTC is the default, need to set c.TimeZone here.

File: cmd/bosun/conf/conf.go (hunk 1 above)
Comment (mjibson) [inline cmd/bosun/conf/conf.go]: This should be a *time.Location

File: cmd/bosun/conf/conf.go [hunk 3]
```diff
@@ -314,6 +315,7 @@ func New(name, text string) (c *Conf, err error) {
 	defer errRecover(&err)
 	c = &Conf{
 		Name:           name,
+                TimeZone:       "UTC",
```
Comment (mjibson) [inline cmd/bosun/conf/conf.go]: time.UTC

File: cmd/bosun/conf/conf.go [hunk 4]
```diff
@@ -363,6 +365,14 @@ func New(name, text string) (c *Conf, err error) {
 func (c *Conf) loadGlobal(p *parse.PairNode) {
 	v := c.Expand(p.Val.Text, nil, false)
 	switch k := p.Key.Text; k {
+        case "timeZone":
+                _, err := time.LoadLocation(v)
```
Comment (mjibson) [inline cmd/bosun/conf/conf.go]: Preserve the value of this and set it as c.TimeZone

File: cmd/bosun/conf/conf.go [hunk 5]
```diff
@@ -363,6 +365,14 @@ func New(name, text string) (c *Conf, err error) {
...
+                if err != nil {
+                        c.error(err)
+                        c.TimeZone = "UTC"
```
Comment (mjibson) [inline cmd/bosun/conf/conf.go]: This is already the case and can be removed.

Comment (wkshare) [inline cmd/bosun/conf/conf.go]: Let me adjust:)  thanks for your suggest.

File: cmd/bosun/conf/conf.go (hunk 4 above)
Comment (wkshare) [inline cmd/bosun/conf/conf.go]: Emm...  Modify and PR.

**Comment by wkshare** [comment]
Excuse me ：）

**Comment by wkshare** [comment]
@mjibson Setting env TZ, This project all use Local.

**Comment by wkshare** [comment]
@mjibson  test case, Let me to modify?

**Comment by wkshare** [comment]
@mjibson give me suggest, Please :)

**Comment by mjibson** [comment]
This PR won't be merged because the implementation is too flawed. For example, there is a setting to configure the timezone, but it is then not used, and instead just uses local everywhere always.

**Pull request closed without merging**